editor/gui/editor_file_dialog.cpp

```diff
@@ -1619,6 +1619,7 @@ void EditorFileDialog::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_thumbnail_result"), &EditorFileDialog::_thumbnail_result);
 	ClassDB::bind_method(D_METHOD("set_disable_overwrite_warning", "disable"), &EditorFileDialog::set_disable_overwrite_warning);
 	ClassDB::bind_method(D_METHOD("is_overwrite_warning_disabled"), &EditorFileDialog::is_overwrite_warning_disabled);
+	ClassDB::bind_method(D_METHOD("add_side_menu", "menu", "title"), &EditorFileDialog::add_side_menu, DEFVAL("Settings:"));
```

Comment:
I actually meant that the title should probably be completely optional. Like, you may want to not have it at all. Given that we can create any sidebar control, I think that would be reasonable, while allowing some standardized title for us to rely on.

Reply:
Sure, we can do that.